### DISCUSSION TITLE
Non-ascii tag query

### DISCUSSION THREAD

**Discussion by arkoort** [srid/neuron#552] [General]
I'v tried:

- `[[z:zettels?tag=тег]]`
- <code>[[z:zettels?tag=&amp;#1090;&amp;#1077;&amp;#1075;]]</code>

In both cases I've got useless link instead of titled list of zettels tagged "тег". Is there a way to get a list of zettels tagged with non-acii symbols?

Of course, `neuron` was launched with `LANG=C.UTF-8`.

**Comment by srid** [comment]
This is a limitation of `z:zettels` queries, them being URIs, which must be properly encoded. See here for a workaround https://github.com/srid/neuron/issues/446#issuecomment-720001775

It is for reasons like this (and for simplicity reasons) why I personally move away from tags to using "folgezettel tagging" (i.e., `#[[тег]]`). Wiki-links don't have that limitation.

**Reply by arkoort** [reply]
Thank you. Only now I fully realized the idea of `#[[link]]`. Now I'll try to break the patterns. :)
